<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditRelationDialog</class>
 <widget class="QDialog" name="EditRelationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>511</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btnSetRelationName">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetCardinalityEnd">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSetCardinalityStart">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>210</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>SET</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemoveRelation">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>260</y>
     <width>241</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>REMOVE RELATION</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>258</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Relation name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtRelationName"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>190</y>
     <width>258</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Cardinality by relation start object</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtCardinalityStart"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>258</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Cardinality by relation end object</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="txtCardinalityEnd"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
